docs/resources/infoblox_ipv4_network_container.md

```diff
@@ -32,5 +36,15 @@ resource "infoblox_ipv4_network_container" "v4net_c2" {
   })
 }
 
-// so far, we do not support dynamic allocation of network containers
+// full set of parameters for dynamic allocation of network containers
+resource "infoblox_ipv4_network_container" "v4net_c3" {
+  parent_cidr = infoblox_ipv4_network_container.v4net_c2.cidr
```

Comment:
Let's use either literal values or a reference to a resource which exists here, on this page.